lib/reports/estimate.rb

```diff
@@ -31,13 +31,12 @@ def cost_report
     def find_matching_ocns(ocns = @ocns)
       ocns.each do |ocn|
         marker.incr
-        cluster = Cluster.find_by(ocns: ocn.to_i,
-          "ht_items.0": {"$exists": 1})
+        cluster = Cluster.for_ocns([ocn.to_i])
         next if cluster.nil?
 
         @num_ocns_matched += 1
-
-        next if clusters_seen.include?(cluster._id)
+        # this will be an issue if data changes while this is running
```

Comment:
> this will be an issue if data changes while this is running

What kind of issue? A cluster gets counted more/less than it should? If so, it's "just" an estimate, does not need to be 100% accurate.

Reply:
Right, overlap for some clusters could get double-counted.

Reply:
I updated the comment to reflect this.